src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -660,7 +660,7 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
             var requestMethodToClrMethodMap = new Dictionary<string, string>(StringComparer.Ordinal);
             var candidateAliases = new Dictionary<string, string>(StringComparer.Ordinal);
 
-            for (TypeInfo t = target.GetType().GetTypeInfo(); t != null; t = t.BaseType?.GetTypeInfo())
+            for (TypeInfo t = target.GetType().GetTypeInfo(); t != null && t != typeof(object).GetTypeInfo(); t = t.BaseType?.GetTypeInfo())
```

Comment:
I prepared a change that did this. It relies on `MethodInfo.GetBaseDefinition()` but that doesn't exist on .netstandard1.1 (it was introduced in netstandard1.5). So rather than have divergent behavior across platforms or re-implementing the method myself (which is non-trivial and arguably impossible to do with 100% accuracy), I'm siding on not implementing it at this point.